vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/network/apis/externalipranger/register.go

```diff
@@ -7,6 +7,16 @@ import (
 
 var GroupVersion = schema.GroupVersion{Group: "network.openshift.io", Version: runtime.APIVersionInternal}
 
+// Kind takes an unqualified kind and returns back a Group qualified GroupKind
+func Kind(kind string) schema.GroupKind {
```

Comment:
put this in the external version

Reply:
not sure i understand your comment. can you please elaborate?